sdk/storage/Azure.Storage.Blobs/src/BlobExtensions.cs

```diff
@@ -742,7 +742,7 @@ internal static BlobProperties ToBlobProperties(this ResponseWithHeaders<BlobGet
                 copyId: response.Headers.CopyId,
                 copyProgress: response.Headers.CopyProgress,
                 copySource: response.Headers.CopySource == null ? null : new Uri(response.Headers.CopySource),
-                copyStatus: response.Headers.CopyStatus.GetValueOrDefault(),
```

Comment:
I think we need to continue populating the old property, to avoid breaking current users.

Reply:
The old property is now a facade for the new one. There is only one field to populate, and both will serve the expected value.